### ISSUE TITLE
Null response without an error

### ISSUE THREAD

**Issue by GoldenGamerLP** [CrystalGamesMC/Scaffolding#4]
**Describe the bug**
So when you trying to load up a schematic via Scaffolding.fromFile or via the FileInputStream the response is always null. There is no exception too which would be thrown. BTW my entire completeable future is beeing closed but i have no idea what why and whats caused it...

**To Reproduce**
1. Load up a schematic
2. look via if(schematic != null) if its null
3. Use it in a CompleteableFuture
4. See null response with no error (Schematics are tested and worked fine)

**Expected behavior**
It should throw an error and should load when the schematics are right, load them up.

**Screenshots**
1. 
![image](https://user-images.githubusercontent.com/65727384/148615831-705dc211-45fa-4108-aaab-a44032b74d4a.png)
2. (Debug) 
![image](https://user-images.githubusercontent.com/65727384/148615890-5111a300-50f0-40da-853a-3739d5e797a7.png)
(- Please ask for a full code snippet)

**Desktop (please complete the following information):**
 - Latest Scaffolding version
 - Latest Minestom verison (Not 1.18 branch)
 - Java 17

**Comment by LooFifteen** [comment]
What kind of schematic are you using? MCEdit or Sponge?

**Comment by GoldenGamerLP** [comment]
I tried both types and also renamed them both to .schematic

**Comment by LooFifteen** [comment]
Are you able to send a screenshot of the schematic open in NBTExplorer?

**Comment by GoldenGamerLP** [comment]
1. Schematic: 
![image](https://user-images.githubusercontent.com/65727384/148619741-614e02d9-f07b-474a-9410-119294ba07bb.png)
2. Schematic: 
![image](https://user-images.githubusercontent.com/65727384/148619759-d33bc9ee-f159-45f9-9072-a95b83a62f63.png)
* Not NBT explorer sorry, i cant install it rn

**Comment by LooFifteen** [comment]
Schematic 1 is a Sponge schematic, try loading it using `new SpongeSchematic()#read`. I don't recognise the second one, are you able to send the file for me to check out tomorrow?

**Comment by GoldenGamerLP** [comment]
Yes the first one is Sponge, the second one had a .nbt thing at the end. Ill try the new SpongeSchematic()#read later. Thanks

**Comment by GoldenGamerLP** [comment]
So i created the Schematic by myself but it didnt work either
- 
![image](https://user-images.githubusercontent.com/65727384/148620842-650cb153-2452-45bb-a33c-853f8c2e8a33.png)

**Comment by LooFifteen** [comment]
How exactly did you know it didn't work?

**Comment by GoldenGamerLP** [comment]
> How exactly did you know it didn't work? 

When I read the nbt via fileinputstream, it caused my whole completable future to exit und stop working, even my debug messages after the loop wouldn't print out.

**Comment by LooFifteen** [comment]
I will need the schematics to reproduce this.

**Comment by GoldenGamerLP** [comment]
Yes, i downloaded a random schematic, here it is: https://workupload.com/file/RNfUqkkMXZc

**Comment by GoldenGamerLP** [comment]
So i tried reading the NBT data myself and this error occoured with this code:
Error:
- 
![image](https://user-images.githubusercontent.com/65727384/148664087-d9f428ea-0363-4d07-b2d7-b3ddcb2b9924.png)

Code to reproduce:
- 
![image](https://user-images.githubusercontent.com/65727384/148664093-2bd199a0-8136-4597-89a1-a1d3a327eca3.png)

**Comment by LooFifteen** [comment]
Okay, I'll look into this sometime tomorrow.

**Comment by GoldenGamerLP** [comment]
Thanks ^^

**Comment by LooFifteen** [comment]
The schematic you provided has loaded on my machine, but the block palette is slightly off.

**Comment by GoldenGamerLP** [comment]
Ah okay, I have no more ideas why it isn't loading and exiting my whole completable furutre. Do you maybe have an idea where the error which I send you is coming from?

**Comment by LooFifteen** [comment]
It may be caused by the fact you are not on the 1.18 branch of Minestom.

**Comment by GoldenGamerLP** [comment]
That could be true, I'm using the latest version of the main branch (-SNAPSHOT)

**Comment by GoldenGamerLP** [comment]
Any news on the bug?

**Comment by LooFifteen** [comment]
I'm not sure, I was able to read and build the schematic perfectly fine. I did discover a different bug but it was unrelated to your issue.
